osscameroon/camerdevs
# Add company rating endpoint

This pull request add the GET company rating endpoint

Steps to verify:
- move to the `backend` folder with `cd ./backend`
- run `make start-postrges` to run an initialized database
- run  `make run` to launch the api
- then run the command `curl -X GET localhost:7000/company-ratings` you should see something like this

```
[{"id":1,"company_id":761,"rating":4,"comment":"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Proin risus. Praesent lectus.","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"id":2,"company_id":995,"rating":4,"comment":"Nam ultrices, libero non mattis pulvinar, nulla pede ullamcorper augue, a suscipit nulla elit ac nulla. Sed vel enim sit amet nunc viverra dapibus. Nulla suscipit ligula in lacus.","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"}
....
]
```

- then run the command `curl -X GET localhost:7000/company-ratings/1` you should see something like this

```
{"id":1,"company_id":761,"rating":4,"comment":"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Proin risus. Praesent lectus.","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"}
```

- then run the command `curl -X GET localhost:7000/company-ratings?company_id=763` you should see something like this

```
[{"id":89,"company_id":763,"rating":4,"comment":"Curabitur gravida nis[{"id":89,"company_id":763,"rating":4,"comment":"Curabitur gravida nis
i at nibh. In hac habitasse platea dictumst. Aliquam augue quam, sollicitudin vitae, consectetuer eget, rutrum at, lorem.\n\nInteger tincidunt ante vel ipsum. Praesent blandit lacinia erat. Vestibulum sed magna at nunc commodo placerat.","createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"id":760,"company_id":763,"rating":3,"comment":"Donec diam neque, vestibulum eget, vulputate ut, ultrices vel, augue. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec pharetra, magna vestibulum aliquet ultrices, erat tortor sollicitudin mi, sit amet lobortis sapien sapien non mi. Integer ac neque.","createdat":"0001-01-01T00:00:00Z","updatedat
":"0001-01-01T00:00:00Z"}]
```
